src/praisonai-agents/praisonaiagents/agent/agent.py

```
@@ -5079,6 +5090,148 @@ def _resolve_cli_backend(self, cli_backend):
+                model_runtime_configs=getattr(self, '_model_runtime_configs', None),
+                provider_runtime_configs=getattr(self, '_provider_runtime_configs', None),
+                legacy_cli_backend=getattr(self, '_cli_backend', None)
+            )
+            
+            if result and result.runtime:
+                return result.runtime
+            
+        except Exception:
+            # Silently fall back to non-runtime execution if resolution fails
+            # This ensures backward compatibility
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)